src/sap.ui.commons/src/sap/ui/commons/themes/base/globalErrorHandler.less

```diff
@@ -0,0 +1,24 @@
+.sapMMsgBoxGEH .expandableTextArea {
```

Comment:
Even when prefixed with a UI5-specific CSS class within this selector, you should not introduce an unprefixed CSS class name like "expandableTextArea": the place where you write it to the HTML is at risk of being styled wrongly by CSS which is added by other libraries or the application.